msg/vehicle_local_position.msg

```diff
@@ -28,6 +36,11 @@ uint8 z_reset_counter
 uint8 vxy_reset_counter
 uint8 vz_reset_counter
 
+# Acceleration in NED frame
+float32 ax        # North acceleration in NED earth-fixed frame, (metres/sec^2)
+float32 ay        # East acceleration in NED earth-fixed frame, (metres/sec^2)
+float32 az        # Down acceleration in NED earth-fixed frame, (metres/sec^2)
```

Comment:
Yes. We want it here because the mavlink message LOCAL_POSITION_NED_COV also has it, and an external estimator (especially VIO) is in the best place to estimate accelerations.